README.md

```diff
@@ -1 +1,3 @@
-# Hello-world
\ No newline at end of file
+# Hello-world
+
+I am Humayoon Rafei learning git-hub with Halim jan for the second day 
```

Comment:
thanks for let me join you learning computer programming.

Reply:
you are welcome